tst/EnergyPlus/unit/DXCoils.unit.cc

```diff
@@ -1407,17 +1417,16 @@ namespace EnergyPlus {
 		Real64 const RatedInletAirHumRat( 0.01125 ); // Humidity ratio corresponding to 80F dry bulb/67F wet bulb
 		std::string const CallingRoutine( "DXCoil_ValidateADPFunction" );
 
-		Real64 DesMassFlow = DXCoil( 1 ).RatedAirVolFlowRate( 1 ) * PsyRhoAirFnPbTdbW( StdBaroPress, RatedInletAirTemp, RatedInletAirHumRat, CallingRoutine );
-		Real64 CBF_calculated = CalcCBF( DXCoil( 1 ).DXCoilType, DXCoil( 1 ).Name, RatedInletAirTemp, RatedInletAirHumRat, DXCoil( 1 ).RatedTotCap( 1 ), DesMassFlow, DXCoil( 1 ).RatedSHR( 1 ), true );
+		Real64 CBF_calculated = CalcCBF( DXCoil( 1 ).DXCoilType, DXCoil( 1 ).Name, RatedInletAirTemp, RatedInletAirHumRat, DXCoil( 1 ).RatedTotCap( 1 ), DXCoil( 1 ).RatedAirVolFlowRate( 1 ), DXCoil( 1 ).RatedSHR( 1 ), true );
 
-		EXPECT_NEAR( 0.747472, DXCoil( 1 ).RatedSHR( 1 ), 0.0000001 );
-		EXPECT_NEAR( 0.1012203, CBF_calculated, 0.0000001 );
```

Comment:
Don't particularly like this change in BF from 0.1 to 0.0003, but it's all in how you select total capacity and air volume flow rate. We don't restrict the users choices right up to the saturation curve (i.e., BF=0.000001 is OK and I have seen 1E-6 before) so this change is arbitrary given the total capacity and volume flow rate selected here since barometric pressure was changed.